modules/core/src/main/java/org/apache/synapse/config/xml/EnrichMediatorFactory.java

```diff
@@ -32,8 +38,13 @@
 
 import java.util.Properties;
 
+/**
+ * Factory for {@link EnrichMediator} instances.
+ */
```

Comment:
Let's remove empty new line after all Javadoc comments

Reply:
Those are not introduced by this commit.